src/components/TodoList.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable react/react-in-jsx-scope */
+import TodoNote from "./TodoNote";
+import "../index.css";
+import React, {useState, useEffect, useRef} from "react";
+import {nanoid} from "nanoid";
+import AddNote from "./AddNote";
+
+const TodoList = () => {
+  const getNotesFunction = () => {
+    const savedNotes = localStorage.getItem("react-notes-data");
+    return savedNotes ? JSON.parse(savedNotes) : [];
+  };
+
+  const [notes, setNotes] = useState(getNotesFunction);
```

Comment:
Why is the indenting for these two variables un even? Check to make sure your linter is running before you commit.

Reply:
They're uneven because line 11 is in a function while line 14 is not. My vscode is set to one space per indent.